src/mdio/schemas/v1/dataset_serializer.py

```diff
@@ -228,12 +218,14 @@ def to_xarray_dataset(mdio_ds: Dataset) -> xr_DataArray:  # noqa: PLR0912
         # Create a custom chunk key encoding with "/" as separator
         chunk_key_encoding = V2ChunkKeyEncoding(separator="/").to_dict()
         encoding = {
-            # NOTE: See Zarr documentation on use of fill_value and _FillValue in Zarr v2 vs v3
-            "_FillValue": _get_fill_value(v.data_type),
             "chunks": chunks,
             "chunk_key_encoding": chunk_key_encoding,
             "compressor": _convert_compressor(v.compressor),
         }
+        if v.name not in no_fill_var_names:
```

Comment:
```suggestion
        if dtype.fields is not None:
```

I think this will allow us to get away the fill value issue without requiring us to pass `no_fill_var_names`, unless there was another reason to allow omitting fill values.